programs/quarter_board/communication.c

```diff
@@ -0,0 +1,163 @@
+#include "ch.h"
+#include "hal.h"
+
+#include <icu_lld.h>
+#include "chprintf.h"
+#include <inttypes.h>
+
+#include "communication.h"

```

Comment:
Please reorder the includes here so that those using `<filename>` notation are separate from those using `"filename"` one, as it changes context and can cause confusion in readers.